packages/web/app/src/components/target/explorer/object-type.tsx

```diff
@@ -34,7 +34,9 @@ export function GraphQLObjectTypeComponent(props: {
   const router = useRouter();
   const searchObj = router.latestLocation.search;
   const search =
-    'search' in searchObj && typeof searchObj.search === 'string' ? searchObj.search : undefined;
+    'search' in searchObj && typeof searchObj.search === 'string'
+      ? searchObj.search.toLowerCase()
```

Comment:
I modified the search behavior wherever I saw it, so that it's not case sensitive. Having a case sensitive search is typically very confusing UX